hedera-mirror-rest/utils.js

```diff
@@ -299,11 +307,14 @@ const getIntegerParam = (param, limit = undefined) => {
 /**
  * Parse the query filer parameter
  * @param paramValues Value of the query param after parsing by ExpressJS
- * @param {Function} processOpAndValue function to compute partial sql clause and sql params using comparator and value
+ * @param {Function} processValue function to sql params using comparator and value
```

Comment:
```suggestion
 * @param {Function} processValue function to extract sql params using comparator and value
```
Reads like there's a missing work here

Reply:
Good catch, fixed